docs/HowToFAQ/FAQModuleImport.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Why is my import opened statement not working?
```

Comment:
Assuming that people will scan through a list of FAQ titles, I'm not sure that this title will help users find this Q&A.  Maybe "Why do children modules not inherit the imports of their parents?"

(There's nothing specific to `opened` here, is there?

Reply:
Reworded the question.
And I'll make a note to go through the whole list for similar overly specific questions.